test/language/expressions/arrow-function/arrow/CoverParenthesizedExpressionAndArrowParameterList-expression-1.js

```diff
@@ -0,0 +1,9 @@
+// Copyright 2015 Microsoft Corporation. All rights reserved.
+// This code is governed by the license found in the LICENSE file.
+
+/*---
+description: "CoverParenthesizedExpressionAndArrowParameterList :( Expression )"

```

Comment:
CoverParenthesizedExpressionAndArrowParameterList : ( Expression ) is not an arrow function grammar production, but an expression production.

For arrow functions CoverParenthesizedExpressionAndArrowParameterList is reinterpreted as ( StrictFormalParameters )

What is this test specifically testing for?